autorest/azure/azure.go

```diff
@@ -50,8 +51,8 @@ type RequestError struct {
 
 // Error returns a human-friendly error message from service error.
 func (e RequestError) Error() string {
-	return fmt.Sprintf("azure: Service returned an error. Code=%q Message=%q Status=%d",
-		e.ServiceError.Code, e.ServiceError.Message, e.StatusCode)
+	return fmt.Sprintf("autorest/azure: Service returned an error. %v Status=%v",
```

Comment:
Hmm %v is going to be super long, you realize that right? Why not `Status=%d Error=%v`?